docs/source/run/parameters.rst

```diff
@@ -504,25 +482,114 @@ which are valid only for certain beam types, are introduced further below under
 
-* ``<beam name> or beams.initialize_on_cpu`` (`bool`) optional (default `0`)
-    Whether to initialize the beam on the CPU instead of the GPU.
-    Initializing the beam on the CPU can be much slower but is necessary if the full beam does not fit into GPU memory.
+Option: ``fixed_weight_pdf``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* ``<beam name>.num_particles`` (`int`)
+    Number of constant weight particles to generate the beam.
+
+* ``<beam name>.pdf`` (`float`)
+    Probability density function to define the longitudinal profile of the beam
+    (the transverse profile is Gaussian). This is a parser function of z
+    that should return the total amount of charge present on each transverse beam slice.
+    This will also be proportional to the per-slice peak density if ``<beam name>.position_mean`` is constant along z,
+    if not the pdf can be calculated from the desired peak density using ``<beam name>.pdf = <peak_density(z)> * <position_std_x(z)> * <position_std_y(z)>``.
+    The probability density function is automatically normalized so that all constant pre factors
+    can be omitted when specifying the function.
+    Examples (replace `z_center`, `z_std`, `z_length`, `z_slope`, `z_min` and `z_max`):
+    Gaussian: ``exp(-0.5*((z-z_center)/z_std)^2)``
+    Cosine: ``(cos(2*pi*(z-z_center)/z_length)+1)*(2*abs(z-z_center)<z_length)``
+    Trapezoidal: ``(z<z_max)*(z>z_min)*(1+z_slope*z)``
+
+* ``<beam name>.total_charge`` (`float`)
+    Total charge of the beam. Note: Either ``total_charge`` or ``density`` must be specified.
+    Only available when running in SI units.
+    The absolute value of this parameter is used when initializing the beam.
+    Note that in contrast to the ``fixed_weight`` injection type, using ``<beam name>.radius`` or
+    a special pdf to emulate `z_min` and `z_max` will result in beam particles being redistributed to
+    other locations and not in them being deleted in the injection, meaning that the resulting beam will
+    have exactly the total charge that is specified here.
+
+* ``<beam name>.density`` (`float`)
+    Peak density of the beam. Note: Either ``total_charge`` or ``density`` must be specified.
+    The absolute value of this parameter is used when initializing the beam.
+    Note that this is the peak density of the analytical profile specified by the `pdf`, `position_mean` and
+    `position_std`, within the limits of the resolution of the numerical evaluation of the pdf. The actual
+    resulting beam profile consists of randomly distributed particles and will likely feature density
+    fluctuations exceeding the specified peak density.
+
+* ``<beam name>.position_mean`` (2 `float`)
+    The mean position of the beam in ``x, y``, separated by a space. Both values can be a function of z.
+    To generate a tilted beam use
+    ``<beam name>.position_mean = "x_center+(z-z_ center)*dx_per_dzeta" "y_center+(z-z_ center)*dy_per_dzeta"``.
```

Comment:
Maybe it was on purpose, but I am not sure why, so could you check whether there should be a space after the underscores?
```suggestion
    ``<beam name>.position_mean = "x_center+(z-z_center)*dx_per_dzeta" "y_center+(z-z_center)*dy_per_dzeta"``.
```